### ISSUE TITLE
ORLEN Paczka, od tygodnia pojawia się komunikat: Opcja 'Inne dane na etykiecie' nie są dostępne dla wybranego przewoźnika

### ISSUE THREAD

**Issue by FishBeat** [allegro/allegro-api#9414]
Wszystko do tej pory było dobrze, aż do ostatniego tygodnia zaczął się pojawiac komunikat:

Opcja 'Inne dane na etykiecie' nie są dostępne dla wybranego przewoźnika

Nic nie było zmieniane w strukturze przesyłanych danych więc nie rozumiem skąd nagle ten błąd. Myślałem, że chodzi o pole DESPRIPTION, ale jak przekażę NULL to nic się nie zmienia, wklejam przykładową strukturę danych:

$data = ["commandId" => NULL, "input" => ["deliveryMethodId" => "04c33651-bd96-484b-a89b-06a797d8eff4",
                              "credentialsId" => NULL,
                              "sender" => ["name" => (string)$firma_imieNazwisko,
                                           "company" => (string)$firma_nazwaLong,
                                           "street" => (string)$firma_adres,
                                           "streetNumber" => "0", 
                                           "postalCode" => (string)$firma_postcode,
                                           "city" => (string)$firma_city,
                                           "countryCode" => "PL",
                                           "email" => "aukcje@xxx.pl",
                                           "phone" => (string)$firma_telefon,
                                           "point" => NULL],
                             "receiver" => ["name" =>  iconv('ISO-8859-2', 'UTF-8', $nazwa),
                                           "company" =>  iconv('ISO-8859-2', 'UTF-8', $nazwa_2),
                                           "street" => iconv('ISO-8859-2', 'UTF-8', $adres),
                                           "streetNumber" => "0", 
                                           "postalCode" => $adres_2kod,
                                           "city" => iconv('ISO-8859-2', 'UTF-8', $adres_2miejscowosc),
                                           "countryCode" => "PL",
                                           "email" => $mail,
                                           "phone" => str_replace(' ','',$telefon),
                                           "point" => $paczkomat],
                              "pickup" => ["name" => (string)$firma_imieNazwisko,
                                           "company" => (string)$firma_nazwaLong,
                                           "street" => (string)$firma_adres,
                                           "streetNumber" => ".", 
                                           "postalCode" => (string)$firma_postcode,
                                           "city" => (string)$firma_city,
                                           "countryCode" => "PL",
                                           "email" => "aukcje@xxx.pl",
                                           "phone" => (string)$firma_telefon,
                                           "point" => "LG-723268-B1-11"],
                             "referenceNumber" => $numer_referencyjny,
                             "description" => "NICK ALLEGRO: $nick_allegro | TOWAR: ".iconv('ISO-8859-2', 'UTF-8', $towar)." ".iconv('ISO-8859-2', 'UTF-8', $marka)."",
                             "packages" => [["type" => "PACKAGE",
                                           "length" => ["value" => "28",
                                                        "unit" => "CENTIMETER"],
                                           "width" => ["value" => "22",
                                                        "unit" => "CENTIMETER"],
                                           "height" => ["value" => "10",
                                                        "unit" => "CENTIMETER"],
                                           "weight" => ["value" => "1",
                                                        "unit" => "KILOGRAMS"]]],
                             "insurance" => ["amount" => number_format($cena_razem,2,'.',' '),
                                            "currency" => "PLN"],
                             "cashOnDelivery" => NULL,
                             "labelFormat" => "PDF"]];

Z góry dziękuję za wszelką pomoc.

**Comment by Lukasz-Zurek** [comment]
@FishBeat Prześlij proszę trace-id, sprawdzę dokładną przyczynę.

**Comment by FishBeat** [comment]
Trace_id: ed251ff4faae21d7
Wysyłka miała dotyczyć Automaty Orlen Paczka

**Comment by Lukasz-Zurek** [comment]
@FishBeat Dane zawarte w polu "Sender" powinny być zgodne z "Pickup" - z tego co widzę różnią się one m.in. wartościami w **"point"** i **"streetNumber"**.

**Comment by FishBeat** [comment]
Tak, już poprawiłem i działa, dziękuję!! Komunikat błędu jest moim zdaniem nieco mylący i bez waszej pomocy nie poradził bym sobie. Dziwne, że wszystko do tej pory działało poprawnie.

**Comment by Lukasz-Zurek** [comment]
@FishBeat Do usług :) Co do komunikatu to masz rację i zwrócę na to uwagę zespołowi odpowiedzialnemu za usługę WZA. Dziękuję.
